element-hq/element-x-android
# feat(knock requests) : branch logic for handling knock requests

## Content
Branch all the logic when displaying banner and knock requests list : 
- allow to dismiss the banner for the currently seen requests
- allow to accept/decline requests and ban users
- everything is live thanks to the sdk


## Motivation and context
https://github.com/element-hq/element-x-android/issues/3692


## Screenshots / GIFs



## Tests



- Create a knockable room
- Open the room
- With another account (on the same HS) navigate to this room (through permalink or room directory)
- Use the knock button
- See the request from the admin account, and play with the actions

## Tested devices

- OS version(s):

## Checklist
